doc/configure-device.md

```diff
@@ -87,24 +87,16 @@ You need a Linux PC (for flashing), a Raspberry Pi 3 or 4 board, and a micro SD
    End /dev/sdc format
    ```
 
-1. Download a Tizen OS image. Visit the links below and download two `.tar.gz` files (for boot and platform). Choose Tizen 6.5 (arm) if you are not sure which image to download.
+1. Download a Tizen OS image. Visit the links below and download two `.tar.gz` files (boot and headed).
 
-   | Tizen version | Image type | Latest image |
-   |-|-|-|
-   | Tizen 6.0 | Pi 3 (arm) | [iot-boot-armv7l-rpi3](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-armv7l-rpi3)<br>[iot-headed-3parts-armv7l-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-armv7l-rpi) |
-   | | Pi 3 (arm64) | [iot-boot-arm64-rpi3](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-arm64-rpi3)<br>[iot-headed-3parts-aarch64-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-aarch64-rpi) |
-   | | Pi 4 (arm) | [iot-boot-armv7l-rpi4](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-armv7l-rpi4)<br>[iot-headed-3parts-armv7l-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-armv7l-rpi) |
-   | | Pi 4 (arm64) | [iot-boot-arm64-rpi4](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-boot-arm64-rpi4)<br>[iot-headed-3parts-aarch64-rpi](http://download.tizen.org/snapshots/tizen/6.0-unified/latest/images/standard/iot-headed-3parts-aarch64-rpi) |
-   | Tizen 6.5 | Pi 3 (arm) | [tizen-boot-armv7l-rpi3](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-boot-armv7l-rpi3)<br>[tizen-headed-armv7l](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-headed-armv7l) |
-   | | Pi 3 (arm64) | [tizen-boot-arm64-rpi3](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-boot-arm64-rpi3)<br>[tizen-headed-aarch64](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-headed-aarch64) |
-   | | Pi 4 (arm) | [tizen-boot-armv7l-rpi4](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-boot-armv7l-rpi4)<br>[tizen-headed-armv7l](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-headed-armv7l) |
-   | | Pi 4 (arm64) | [tizen-boot-arm64-rpi4](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-boot-arm64-rpi4)<br>[tizen-headed-aarch64](http://download.tizen.org/snapshots/tizen/6.5-unified/latest/images/standard/tizen-headed-aarch64) |
+   - Pi 3: [tizen-boot-arm64-rpi3](http://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-boot-arm64-rpi3) / [tizen-headed-aarch64](http://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-headed-aarch64)
```

Comment:
http://download.tizen.org/releases/milestone/TIZEN/Tizen/Tizen-Unified/latest/images/standard/tizen-boot-arm64-rpi3
This link is broken.

Reply:
Oops. They removed the RPi3 support on the day after creating this PR.